deeplearning4j-nearestneighbors-parent/deeplearning4j-nearestneighbors-model/src/main/java/org/deeplearning4j/nearestneighbor/model/NearestNeighborsResult.java

```diff
@@ -12,5 +11,5 @@
 @NoArgsConstructor
 public class NearestNeighborsResult {
```

Comment:
So, as a hack that would work, except as you know, we also have a `NearestNeighborResults` class for a batch of those. Wouldn't it be more logical to have the distance added in `NearestNeighborResults` as a field ? We'd have the distance added in at the list's creation, and would just have `NearestNeighborResult` be a pass-through construction that just unfolds the tuple a list element contains.